README.md

```diff
@@ -21,7 +21,8 @@ account.
 Never heard of Interledger before, or you would like to learn more? Here are some good places to start:
 
 - [Good first issues](https://github.com/interledger/rafiki/contribute)
-- [Interledger](https://interledger.org/)
+- [Interledger Explainder Video](https://twitter.com/Interledger/status/1567916000074678272)
```

Comment:
```suggestion
- [Interledger Explainer Video](https://twitter.com/Interledger/status/1567916000074678272)
```

Reply:
Nice catch. I ran Grammarly over it before committing, you were better than Grammarly!